packages/design-tokens/src/js/themes/theme-light-default/colors.ts

```diff
@@ -151,6 +151,12 @@ export const formFieldWarning = '#c26400';
 export const gradientBasicOverlayColor01 = '#fff';
 export const gradientBasicOverlayColor02 = '#fff0';
 
+export const gradientPrimaryColor01 = '#7847ff';
+export const gradientPrimaryColor02 = '#482b99';
```

Comment:
```suggestion
export const gradientPrimaryColorStart = '#7847ff';
export const gradientPrimaryColorEnd = '#482b99';
```

?

Reply:
Nope, please. This will be generated, and there could be any number of colours.